### ISSUE TITLE
Map on item frame are bugged in some cases

### ISSUE THREAD

**Issue by zeid0ne** [FiniteReality/embeddium#282]
### Bug Description

When i have embeddium installed, sometime some map rendered on an item frame are bugged.
The issue invole transparence.

### Reproduction Steps

Create a new world.
Make a map (defaut scale dezoomed one time), above a biome with a lot of glace.
Put it on a item frame.
Look at it from different angles.

### Log File

On the video it's a glowing item frame but a simple item frame has the same bug.

https://www.youtube.com/watch?v=QaXZJ1_UBlg&feature=youtu.be

**Comment by embeddedt** [comment]
Can you attach a world ZIP that showcases this bug (ideally one that only requires Embeddium installed)? Having trouble recreating it locally.

**Comment by zeid0ne** [comment]
I'll do it next week

**Comment by RemasteredArch** [comment]
I used to have a similar issue in the modpack (1.20.1 Fabric, ~101 mods) I play with. However, sometime in the last couple of months, it seems to have been fixed by the pack either adding, removing, or updating mods. I haven't tested it to be sure, but I'm fairly certain it was caused by Moonlight updating because that's what I had previously found to be the cause.

Looks like your install has other mods, so try updating your [Moonlight](https://modrinth.com/mod/moonlight) version if that's installed (and presumably update [Supplementaries](https://modrinth.com/mod/supplementaries)/[Admendments](https://modrinth.com/mod/amendments) as well, if that's why you have it).

For posterity, I looked into my testing from late October 2023, where I replicated this bug with:
- Minecraft 1.20.1
- Fabric Loader 14.24
- Fabric API 0.90.4
- Moonlight 2.8.52
- Nvidia GeForce 3070 @ Game Ready Drivers 545.92
    - This issue did not occur on my laptop, Intel Core i5-1135G7 w/ Iris Xe graphics

However, the bug continued through several modpack updates until sometime recently.
